R/dal.R

```diff
@@ -152,8 +160,8 @@ edav_io <- function(
       stop("File does not exist")
     }
 
-    if (!grepl(".rds|.rda|.csv", file_loc)) {
-      stop("At the moment only 'rds' 'rda' and 'csv' are supported for reading.")
+    if (!grepl(".rds|.rda|.csv|.xlsx", file_loc)) {
```

Comment:
Is .xls also supported? I see it in the list at the bottom and wanted to make sure they were flagged similarly